routes/database.rb

```diff
@@ -104,15 +104,13 @@ def self.registered(app)
                   server:    params[:server] || db.server,
                   modified:   DateTime.now || db.modified,
                   active: params[:active] || db.active)
-        redirect "/db/#{params[:id]}"
-      end
 
-      app.delete '/db/:id/?' do
```

Comment:
as with clients, I think we want an actual delete method here, but in this context (calling delete on a database), we don't need to re-assign it, we just want to deactivate it. Re-assigning is only needed if we've deleted the parent project. Same comment regarding nodes.